tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj

```diff
@@ -31,6 +31,16 @@
     <PackageReference Include="log4net" Version="2.0.14" Condition="'$(TargetFramework)' == 'net6.0'" />
     <PackageReference Include="log4net.Ext.Json" Version="2.0.10.1" Condition="'$(TargetFramework)' == 'net6.0'" />
 
+    <!-- MySql.Data framework references -->
+    <PackageReference Include ="MySql.Data" Version="6.10.7" Condition="'$(TargetFramework)' == 'net462'" />
+    <PackageReference Include ="MySql.Data" Version="6.10.9" Condition="'$(TargetFramework)' == 'net471'" />
```

Comment:
Is there a reason why the "middle" version is 6.10.9 for framework and 8.0.15 for core?

Reply:
Made both "middle" versions 8.0.15 for framework and core.